dhis-2/dhis-support/dhis-support-test/src/main/java/org/hisp/dhis/DhisConvenienceTest.java

```diff
@@ -2111,7 +2129,7 @@ protected User createUserAndInjectSecurityContext( Set<OrganisationUnit> organis
         }
 
         UserAuthorityGroup userAuthorityGroup = new UserAuthorityGroup();
-        userAuthorityGroup.setName( "Superuser" );
+        userAuthorityGroup.setName( "DhisConvenienceTestUserRole" );

```

Comment:
Can this lead to other tests failing? Maybe tests checking for logged in user?

Reply:
Sure, don't think so based on running tests, but I can revert to avoid unnecessary risk.